.github/workflows/ci.yaml

```diff
@@ -503,6 +527,11 @@ jobs:
       with:
         name: ${{ needs.python_sdist.outputs.sdist_artifact_name }}
 
+    - name: install python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.11
```

Comment:
In order to postpone future required updates, what do you think about increasing this to the latest supported minor version?

Reply:
sure, done